routes/personal/personal.controller.js

```diff
@@ -9,6 +9,14 @@ module.exports = function(app) {
   app.get('/personal/address/edit', getAddressEdit)
   app.post('/personal/address/edit', validateRedirect, checkSchema(addressSchema), postAddress)
 
+  app.get('/personal/residence', (req, res) => res.render('personal/residence'))
+  app.post(
+    '/personal/residence',
+    checkSchema(residenceSchema),
+    validateRedirect,
```

Comment:
Let's move this in front of the schema check. If the `redirect` isn't there, we should fail that first before checking the schema at all.